### ISSUE TITLE
Cannot compile example

### ISSUE THREAD

**Issue by kper** [mattnenterprise/rust-imap#79]
I encountered some troubles when I wanted to implement `imap` in my project. 

```
error[E0308]: mismatched types
  --> src/server.rs:36:97
   |
36 |         let mut imap_socket = imap::client::Client::secure_connect(socket_addr, &master.domain, &ssl_connector).unwrap();
   |                                                                                                 ^^^^^^^^^^^^^^ expected struct `native_tls::TlsConnector`, found a different struct `native_tls::TlsConnector`
   |
   = note: expected type `&native_tls::TlsConnector` (struct `native_tls::TlsConnector`)
              found type `&native_tls::TlsConnector` (struct `native_tls::TlsConnector`)
note: Perhaps two different versions of crate `native_tls` are being used?
```

**I fixed it by downgrading the `native_tls` dependency to version `0.1`.** 
So my `Cargo.toml` looks like this now:
```
imap = "0.8.1"
native-tls = "0.1"
```

Maybe you could make the `native_tls` dependency of imap public.

**Comment by jonhoo** [comment]
Ah, sorry, this has been fixed in #73, but because of #69 I cannot yet publish a new version of the crate to make that change visible (which would solve your problem) :(

ping @mattnenterprise

**Comment by jonhoo** [comment]
Fixed in `0.9`!
